Update GUILD Novatos / Eventpoint

danielgj opened the pull request:
Actualizada la info del Guild de novatos en lo relativo al proyecto Eventpoints

danielgj committed:
Update GUILD Novatos / Eventpoint

KoolTheba commented:
Hola @danielgj ! Gracias por completar la info! Añade please un poco más: estado del proyecto y tecnologías que estáis usando, aunque en vuestro repo se pudiese ver, gracias de nuevo!!! ;) 
Luego mergeo...

danielgj committed:
Actualización del Proyecto EventPoints

Se ha incluido el listado de tecnologías así como el estado actual del proyecto

danielgj commented:
Buenas!

He actualizado en mi repo y creo que automáticamente se ha actualizado el
PR. Aunque no lo tengo claro...

Saludos!

El dom., 14 oct. 2018 a las 21:53, Theba Gomez (<notifications@github.com>)
escribió:

> Hola @danielgj <https://github.com/danielgj> ! Gracias por completar la
> info! Añade please un poco más: estado del proyecto y tecnologías que
> estáis usando, aunque en vuestro repo se pudiese ver, gracias de nuevo!!! ;)
> Luego mergeo...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OSWeekends/Guilds/pull/59#issuecomment-429656879>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AI_BMQF3FgsLKj7mk9bS0XiLFLfXev5Hks5uk5ZAgaJpZM4XbS_t>
> .
>


-- 
 Daniel García Jones

baumannzone commented on Listado Guilds.md:
```diff
@@ -259,6 +259,36 @@ Canal de SLACK: **#pr_read4kids_kidsgld**
+- @Albametnet
+- @lucbpz
+- @carlosfdev
+- @baumannzone
```
🏆

KoolTheba commented:
@danielgj ! Ricardo subió la info completa ;) Cancelo este tuyo pero muuuuchas gracias !!
